tools/param-name-importer/param-name-importer.csproj

```diff
@@ -9,10 +9,16 @@
   <ItemGroup>
     <!-- Not sure why needed, but only System.IO.Compression.FileSystem.dll is
       included by default, and ZipArchive is type forwarded to System.IO.Compression -->
-    <Reference Include="System.IO.Compression" />
+    <Reference Include="System.IO.Compression" Condition="'$(TargetFramework)' == 'net472'" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Xml.SgmlReader" Version="1.8.14" />
+    <!-- This package erroneously contains /netcoreapp3.1/SgmlReader.exe and /netcoreapp3.1/SgmlReaderDll.dll.
+         We are going to use a package reference to download the nuget, and a regular reference to actually
+         reference the correct assembly. -->
+    <PackageReference Include="Microsoft.Xml.SgmlReader" Version="1.8.16" ExcludeAssets="Compile" GeneratePathProperty="true" />
```

Comment:
Does `@(PackageReference)` supports `%(ReferenceOutputAssembly)` metadata, e.g.

```xml
<PackageReference Include="Microsoft.Xml.SgmlReader" Version="1.8.16" ExcludeAssets="Compile" GeneratePathProperty="true" ReferenceOutputAssembly="false" />
```

If it does, that might make it clearer that the `@(PackageReference)` is present *only* for download purposes, and not for `csc /reference` purposes.

Reply:
I think `%(ReferenceOutputAssembly)` is only supported on `@(ProjectReference)`.  NuGet packages have no concept of output assemblies, they're just a zip of files in magic directories. The `ExcludeAssets="Compile"` is what is telling `MSBuild` not to use the references for `csc`.